src/main/resources/com/cloudbees/plugins/credentials/Messages.properties

```diff
@@ -71,7 +71,7 @@ SystemCredentialsProvider.ProviderImpl.DisplayName=Jenkins Credentials Provider
 SystemCredentialsProvider.UserFacingAction.DisplayName=System
 UserCredentialsProvider_DisplayName=Credentials
 UserCredentialsProvider.UserFacingAction.DisplayName=User
-GlobalCredentialsConfiguration.DisplayName=Configure Credentials
+GlobalCredentialsConfiguration.DisplayName=Configure Credentials Providers
```

Comment:
```suggestion
GlobalCredentialsConfiguration.DisplayName=Configure Credential Providers
```

Reply:
Not sure, They are called 'Credentials' everywhere

Reply:
I mentioned this because the subtext in the screenshot shows `Credential Providers`